### ISSUE TITLE
Change "Latest contributions" to use sentence text as click target

### ISSUE THREAD

**Issue by githubshrek** [Tatoeba/tatoeba2#2781]
Currently the https://tatoeba.org/eng/contributions/latest page uses the "i" icon at the end of the sentence to link to the sentence page. I propose that we:

- use the sentence text itself to do the link (using the same Tatoeba-green link colour and hover behaviour as the link to the user profile)
- remove the "i" icon, which will then become redundant

![image](https://user-images.githubusercontent.com/7943077/117635842-84fc7580-b180-11eb-97be-620e37bf9895.png)

This is the same concept as expressed in https://github.com/Tatoeba/tatoeba2/issues/2779 but explicitly making the change here on the "Latest contributions" page. Indeed, this would be a gentle way to introduce the idea of linking based on the sentence text.

**Comment by githubshrek** [comment]
I've implemented this locally and I think it works very well.

Home page:
![image](https://user-images.githubusercontent.com/7943077/118228438-7a94f100-b48a-11eb-85bf-7c1eac7b59de.png)

Latest contributions page:
![image](https://user-images.githubusercontent.com/7943077/118228385-63560380-b48a-11eb-9b93-27228c85af58.png)

**Comment by githubshrek** [comment]
For reference documentation (mine more than anyone else's), the following are the key files and changes to achieve this:

# src/Template/Element/latest_contributions.ctp
This file is the shell around the Latest Contributions and did not need changing. (The shocking pink is just some local testing.)
![image](https://user-images.githubusercontent.com/7943077/118301747-b7410680-b4e3-11eb-9898-21765c2d6aa8.png)

# src/Template/Contributions/latest.ctp
Also part of the shell and did not need changing.

# src/Template/Element/logs/log_text.ctp
This contains the sentence itself. It also contains the "infoLabel" (e.g. "added by admin, 9 days ago")
![image](https://user-images.githubusercontent.com/7943077/118300635-74cafa00-b4e2-11eb-8d69-289b671205bd.png)

# src/Template/Element/logs/log_entry.ctp 
This contains the "info" icon, which is no longer needed.
![image](https://user-images.githubusercontent.com/7943077/118303745-16077f80-b4e6-11eb-85ad-cbf9a998794d.png)

# webroot/css/layouts/elements.css
Changed here the text size of the sentence. The clickable sentence needed (I think) to appear bigger than the default size once it became the clickable target. Original size:
![image](https://user-images.githubusercontent.com/7943077/118302251-464e1e80-b4e4-11eb-97a4-359f7547f02e.png)

Larger size
![image](https://user-images.githubusercontent.com/7943077/118302418-7b5a7100-b4e4-11eb-8be6-7362759155f8.png)

**Comment by jiru** [comment]
Thanks for your suggestion. While I get the UI benefits of using the sentence text as click target, having the whole text in green feels a bit odd to me. Sorry that I can’t really explain the reason, I just feel something’s wrong. It may be just about my sticky habits.

> For reference documentation (mine more than anyone else's), the following are the key files and changes to achieve this

Consider pushing theses changes in your fork instead of using this issue as a memo.

**Comment by githubshrek** [comment]
> Consider pushing theses changes in your fork instead of using this issue as a memo.

Yes, I understand the theory. The main problem is that currently my development environment is not set up properly at all. I have got Imouto setup working with Vagrant, but I don't have a Windows command box which can access it, so my actual git work is going on in a totally different folder. That means that when I've made changes I am happy with in Imouto then I manually copy the files across to the folder with my git tracking! Yes, I know it's a mess, but it's one reason I'm also carefully documenting with screenshots the work I'm doing so that I don't lose track completely.

**Comment by jiru** [comment]
It’s okay :+1:

**Comment by ckjpn** [comment]
Note that in the past the text of a sentence was a clickable link.  However, I think it was thought that the "i" icon would be more intuitive for people and was changed.  I think perhaps that at that time some people also had trouble when they tried to copy the sentence's text from the page.  However, now we have a "copy" button next to each sentence to do this.

Unless there is a good reason to revert this, I think this issue could be closed.
